### DISCUSSION TITLE
Added Device in Z-wave JS UI still showing as new device in HA

### DISCUSSION THREAD

**Discussion by shawnhaywood** [zwave-js/zwave-js-ui#2960] [Q&A] [answered]
I added Z-wave JS UI to home assistant after disabling Z-wave JS. After a bit of work, I successfully added the z-stick Gen 5 controller via Z-wave JS UI and it shows in the control panel.  However, the device is showing as a new Z-wave device in the HA integrations area.  The only option is to configure or ignore the device. What should I do?

**Comment by kpine** [comment, marked as answer]
If this is the USB device you're referring to, just ignore it.

**Reply by shawnhaywood** [reply]
Yes, it is a usb z-wave z-stick Gen5 controller attached to the usb port.

**Reply by kpine** [reply]
That's HA's USB discovery feature. It doesn't know the stick is being used with ZUI. If you attempt to set it up, it will install the official add-on, so it's safe to ignore.

**Reply by shawnhaywood** [reply]
Thank you!
